packages/material-ui/src/useAutocomplete/useAutocomplete.d.ts

```diff
@@ -0,0 +1,16 @@
+export {
+  useAutocomplete as default,
+  AutocompleteChangeDetails,
+  AutocompleteChangeReason,
+  AutocompleteCloseReason,
+  AutocompleteFreeSoloValueMapping,
+  AutocompleteGetTagProps,
+  AutocompleteGroupedOption,
+  AutocompleteHighlightChangeReason,
+  AutocompleteInputChangeReason,
+  AutocompleteValue as Value,
```

Comment:
Would it make sense to fix this export before we go to stable?
```suggestion
  AutocompleteValue,
```

Reply:
I guess the breaking change is not so big, so yeah, why not. Do you think we should create a codemod for this as well?